frontend/src/hooks/useGetTimezones.ts

```diff
@@ -0,0 +1,22 @@
+import TimezoneCountries from 'countries-and-timezones';
+import { useCallback } from 'react';
+import { ISelectOption } from '../component/common/GeneralSelect/GeneralSelect';
+
+export interface ITimezoneSelect extends ISelectOption {
+    utcOffset: string;
+}
+export const useGetTimezones = () => {
```

Comment:
I don't think this needs to be a hook. `const getTimezones = () => ...` or even `const timezones = ` will work the same.